lib/exercism/user_finished_tracks.rb

```diff
@@ -26,11 +43,11 @@ def started_tracks
 
   def completed_count_sql
     <<-SQL
-    SELECT language AS track_id, CAST (COUNT(id) AS INTEGER) AS completed_count
+    SELECT language AS track_id, slug AS problem_slug
```

Comment:
To make this query return something like I mentioned above we would keep the `GROUP BY language` and replace `CAST (COUNT(id) AS INTEGER) AS completed_count` with `string_agg(slug, ',') AS completed_problems`.

@kytrinyx We also have `array_agg` here, but I could not make the `execute` method return a proper array of problems. 😞